client/index.html

```diff
@@ -529,53 +529,8 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
-						<h2>Autocompletion</h2>
-
-						<p>
-							Start typing the following characters followed by any letter to
-							trigger the autocompletion dropdown:
-						</p>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>@</code>
-							</div>
-							<div class="description">
-								<p>Nickname</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>#</code>
-							</div>
-							<div class="description">
-								<p>Channel</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/</code>
-							</div>
-							<div class="description">
-								<p>Commands (see list of commands below)</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>:</code>
-							</div>
-							<div class="description">
-								<p>Emoji</p>
-							</div>
-						</div>
-
 						<h2>Commands</h2>
 
-						<p>All commands can be autocompleted with <kbd>tab</kbd>.</p>
-
```

Comment:
This used to be necessary because nothing on the UI would indicate that commands could be completed. Now, typing `/` opens the autocompletion dropdown, so visual help is making this redundant.